rosdoc2/verbs/build/builders/sphinx_builder.py

```diff
@@ -88,8 +88,9 @@ def ensure_global(name, default):
     }})
 
 if rosdoc2_settings.get('override_theme', True):
+    extensions.append('sphinx_rtd_theme')
+    html_theme = 'sphinx_rtd_theme'
     print(f"[rosdoc2] overriding theme to be '{{html_theme}}'", file=sys.stderr)
```

Comment:
nitpick: the other stanza's have the print before the activity

```suggestion
    print(f"[rosdoc2] overriding theme to be '{{html_theme}}'", file=sys.stderr)
    extensions.append('sphinx_rtd_theme')
    html_theme = 'sphinx_rtd_theme'
```

Reply:
I don't think this works in this case, though, since we need to set the `html_theme` variable before using it in the print.

Reply:
Oh yeah 🙃